src/bin/queues.rs

```diff
@@ -29,10 +29,10 @@ lazy_static! {
         };
         Queues::new::<Sqs>(
             QueueIds {
-                bounce: sqs_urls.bounce.clone(),
-                complaint: sqs_urls.complaint.clone(),
-                delivery: sqs_urls.delivery.clone(),
-                notification: sqs_urls.notification.clone(),
+                bounce: sqs_urls.bounce.0.clone(),
```

Comment:
I don't know if I like adding this `.0` everywhere that we want to access the inner `String` for these new types. Do you have any other solution for this? I could implement `Clone` for the new types easily but that wouldn't fix the whole issue...

Reply:
Although it's a little bit annoying, let's not lose too much sleep over it. Even if we don't come up with something better right now, we might do in the future.